akd/src/utils.rs

```diff
@@ -51,3 +51,24 @@ macro_rules! test_config {
         }
     };
 }
+
+/// Macro used for running tests with different configurations
+#[cfg(any(test, feature = "public_tests"))]
+#[macro_export]
+macro_rules! versioned_test_config {
```

Comment:
This is not used anywhere and can be deleted, right?

Reply:
yes, will delete